pybind11/setup_helpers.py

```diff
@@ -0,0 +1,260 @@
+# -*- coding: utf-8 -*-
+
+"""
+This module provides a
```

Comment:
Waiting for other reviews before finalizing this? Will keep this as a reminder if so.

Reply:
Yes, I think @YannickJadoul is going to go over it again and @wjakob will take a look.